docs/release/trg-2/trg-2-4.md

```diff
@@ -2,29 +2,38 @@
 title: TRG 2.04 - Leading product repository
 ---
 
-| Status | Created     | Post-History    |
-|--------|-------------|-----------------|
-| Active | 07-Mar-2023 |                 |
-|        | 10-Jan-2023 | Initial release |
+| Status | Created     | Post-History                                   |
+|--------|-------------|------------------------------------------------|
+| Update | 12-Dez-2023 | add example leading repository reference links |
+| Active | 07-Mar-2023 |                                                |
+|        | 10-Jan-2023 | Initial release                                |
 
 ## Why
 
-We have multiply stakeholders for a repository: System team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
+We have multiply stakeholders for a repository: system team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
 They need to be able to understand complicated product structures in an intuitive way to have an easier task of diving into a product.
 
 ## Description
 
-Each product **must** have a leading product repository that can point the users to all the parts/components/charts/releases of that product.
+Each product **must** have a leading product repository that points to all the parts/components/charts/releases of the product.
 This can be achieved by having a single repository for a product or creating references to the other repositories.
 
-A good example when a product has a separate repository for frontend and backend. Then the product team **must** have a leading repository where they reference the components/repositories of the product:
+Good examples of products got separate repositories for each component (e.g. frontend and backend).
```

Comment:
Not sure, what this sentence should tell me. I think we don't lose much, if we keep this one out.

Reply:
i think the examples are missing here?